audio/out/ao_alsa.c

```diff
@@ -42,8 +42,11 @@
 
 #include <alsa/asoundlib.h>
 
-#define HAVE_CHMAP_API \
-    (defined(SND_CHMAP_API_VERSION) && SND_CHMAP_API_VERSION >= (1 << 16))
+#if defined(SND_CHMAP_API_VERSION) && SND_CHMAP_API_VERSION >= (1 << 16)
```

Comment:
commit message: 
* "Fix undefine**d** ..."
* Rather than "prints the warning", "this commit eliminates the warning"